File: app/src/main/java/com/jd/app/JDApplication.java
package com.jd.app;


import com.jd.app.plugin.DataAnalysis;
import com.jd.core.base.BaseApplication;
import com.renny.libcore.InitContext;

@InitContext
public class JDApplication extends BaseApplication {

    public void onCreate() {
        super.onCreate();
        DataAnalysis.collectionActivity(this);
    }

}

File: JDApp-Kotlin/app/src/main/java/com/jd/app/JDApplication.kt
package com.jd.app


import com.jd.core.base.BaseApplication
import com.renny.libcore.InitContext

@InitContext
class JDApplication : BaseApplication() {

    override fun onCreate() {
        super.onCreate()
      //  DataAnalysis.collectionActivity(this)
    }

}
